vault/resource_ldap_secret_backend_static_role_test.go

```diff
@@ -43,6 +43,13 @@ func TestAccLDAPSecretBackendStaticRole(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, consts.FieldRotationPeriod, rotationPeriod),
 				),
 			},
+			{
+				SkipFunc: func() (bool, error) {
+					return !testProvider.Meta().(*provider.ProviderMeta).IsAPISupported(provider.VaultVersion116), nil
+				},
+				Config: testLDAPSecretBackendStaticRoleConfig_withSkip(path, bindDN, bindPass, url, username, dn, username, rotationPeriod),
+				Check:  resource.TestCheckResourceAttrSet(resourceName, consts.FieldSkipImportRotation),
```

Comment:
Can we check that this is the value we expect?
```suggestion
				Check:  resource.TestCheckResourceAttr(resourceName, consts.FieldSkipImportRotation, "true"),
```